### PR TITLE
Uploading initial opt model.py file

### LINKED ISSUE #11: Translate initial cost model to python code

### PR DESCRIPTION
**Pull request by AaronDzf**
Model still WIP - only use it as sample.
Create pipeline functions using a helper.py file

### CODE REVIEW

**Review by gaurav613 — COMMENTED** [review]
can you mark the variables that will be assigned to the inputs? it can be done in the PR itself

File: app/routes.py [hunk 1]
```diff
@@ -146,11 +146,15 @@ def renderInputs2():
 
     return render_template('home.html', newelectricity_form=form, scroll=scroll, billtype=billtype)
 
+from .optimization_model import optimize
 @app.route('/renderResults', methods=['GET', 'POST'])
 def render_Results():
     complete_form = request.args['Complete_form']
     scrollto_results = request.args['scroll']
-    dataframe = None
+    result = optimize()
```
Comment (Rolo123y) [inline app/routes.py]: need to pass in the dictionary called 'completed_form' into optimize()

File: app/optimization_model.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+e.setParam('WLSACCESSID', WLSACCESSID)
+e.setParam('WLSSECRET', WLSSECRET)
+e.setParam('LICENSEID', LICENSEID)
+# e.start()
+
+# Read backend data from github
+intensity_scores = pd.read_csv("https://raw.githubusercontent.com/gaurav613/fydp_bess/main/Data/Intensity_score.csv")
+
+"""Variables: Decision Variables and Parameters"""
+def optimize(CN = 0.17, CM = 0.113, CF = 0.082, UN_m = 500, UM_m = 300, UF_m = 380):
```
Comment (Rolo123y) [inline app/optimization_model.py]: need to parse the incoming dictionary and assign correct variables for the model.

**Review by Rolo123y — CHANGES_REQUESTED** [review]

File: app/optimization_model.py (hunk 2 above)
Comment (Rolo123y) [reply app/optimization_model.py]: **time of use** dictionary will look as follows:
image

**Tiered** dictionary will look as follows:
image

**Comment by gaurav613** [comment]
Looks good to me, merging now. Change can be made later.

**Review by Rolo123y — APPROVED** [review]

**Pull request merged**